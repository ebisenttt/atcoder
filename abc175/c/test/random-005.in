218873866792353 438601357911107 611467098671044
