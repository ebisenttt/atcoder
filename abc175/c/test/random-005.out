392593231878691
